### PR TITLE
Updatedatabase

### PR DESCRIPTION
**Pull request by oleksandrlytvyn1223**

### CODE REVIEW

File: gtd-timer/Controllers/SignUpController.cs [hunk 1]
```diff
@@ -43,18 +45,19 @@ public IActionResult GetByID(int id)
...
-            } 
+            }
 
             User user = new User
             {
                 FirstName = model.FirstName,
                 LastName = model.LastName,
                 Email = model.Email,
-                PasswordHash=model.Password
+                PasswordHash = pshash.HashPassword(model.Password)
```
Comment (roman-fay) [inline gtd-timer/Controllers/SignUpController.cs]: as another agreement - don't push unrelated changes in one pull request. 
Pull requests and branches are cheap, and separating different changes saves a lot of headache in the future.

**Comment by oleksandrlytvyn1223** [comment]
Ok i will do that in future. Can you aprove it or i need to divide it on two different requests?

File: gtd-timer/Controllers/SignUpController.cs (hunk 1 above)
Comment (oleksandrlytvyn1223) [reply gtd-timer/Controllers/SignUpController.cs]: Ok i will do that in future. Can you aprove it or i need to divide it on two different requests?

**Comment by oleksandrlytvyn1223** [comment]
Canseled changes about hashing of password

**Comment by roman-fay** [comment]
And as an overall comment - please remove redundant field names like Timer.TimerName, Preset.PresetName etc.

**Comment by roman-fay** [comment]
And another thing - we still have to figure out how to store timers order in database, right?

**Comment by oleksandrlytvyn1223** [comment]
Yeah but we decided that for now it would be in same sequense as we added them and later we will change it

**Comment by oleksandrlytvyn1223** [comment]
@roman-fay  changed all what you said

**Pull request merged**